### ISSUE TITLE
Error writing to file descriptor: Broken pipe

### ISSUE THREAD

**Issue by ktw1525** [GSConnect/gnome-shell-extension-gsconnect#1395]
```
GSConnect: 50 (user)
GJS:       17200
Session:   x11
OS:        Ubuntu 22.04 LTS

get@/home/ktw/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/__init__.js:392:37
_initCertificate@/home/ktw/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/backends/lan.js:182:13
start@/home/ktw/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/backends/lan.js:440:18
_loadBackends@/home/ktw/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/manager.js:255:25
start@/home/ktw/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/manager.js:458:14
vfunc_startup@/home/ktw/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/daemon.js:258:22
vfunc_handle_local_options@/home/ktw/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/daemon.js:664:18
@/home/ktw/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/daemon.js:727:17
```

**Comment by herrminni** [comment]
Hi @ktw1525,
Hi @EfelerGibi,

I had the same problem and the issue was, that the openssl library was not working any more.
To confirm, if you have the same problem, just open a terminal an type `openssl` and see, if the program is running without errors.

In my case, the DELL "command-configure" and "srvadmin-hapi" misconfigured openssl and it was not working any more.

The init function in your stack-trace wants to load the gsconnect certificate and get the name of the certificate and this causes the issue - so no problem with the `gsconnect` extension, but with `openssl`.

In my case, i had to remove those two DELL Debian packages I mentioned above.

The issue #1374 seems to be a duplicate.

**Comment by EfelerGibi** [comment]
> I had the same problem and the issue was, that the openssl library was not working any more. To confirm, if you have the same problem, just open a terminal an type `openssl` and see, if the program is running without errors.

Thanks a lot, apparently my openssl config was broken. I fixed it and now the plugin works! Thanks for the help, even when the issue post was really bad.

**Comment by andyholmes** [comment]
Closing since it seems to have been resolved; discussion can continue in #1209 if necessary.
